CONTRIBUTING.rst

```diff
@@ -233,18 +233,15 @@ run the same tests the CI system runs, follow these steps from within your
 local Zephyr source working directory::
 
     $ source zephyr-env.sh
-    $ make host-tools
-    $ export PREBUILT_HOST_TOOLS=${ZEPHYR_BASE}/bin
-    $ export USE_CCACHE=1
-    $ ./scripts/sanitycheck
+    $ ./scripts/sanitycheck --ccache
```

Comment:
i really hope we didn't purge the USE_CCACHE variable...
It might be better to direct users to that instead of having to pass "--ccache" every time

Reply:
We did. I think I know what kind of behaviour you want. I'll push a fix.

Reply:
Fixed: https://github.com/zephyrproject-rtos/zephyr/pull/4837